pandas/tests/extension/test_arrow.py

```diff
@@ -1261,8 +1209,9 @@ def test_invalid_other_comp(self, data, comparison_op):
 
 
 def test_arrowdtype_construct_from_string_type_with_unsupported_parameters():
-    with pytest.raises(NotImplementedError, match="Passing pyarrow type"):
-        ArrowDtype.construct_from_string("timestamp[s, tz=UTC][pyarrow]")
```

Comment:
Could we keep this test but with a type (possibly fake) so we still test this exception?